stable/postgresql/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: postgresql
-version: 0.15.1
+version: 0.15.2
```

Comment:
new feature is added - make it 0.16.0 instead

Reply:
Done.